src/dependency-manager/src/service-config/v1.ts

```diff
@@ -72,6 +72,11 @@ export class InterfaceSpecV1 extends BaseSpec {
 
   @IsOptional({ always: true })
   url?: string;
+
+  @IsOptional({ always: true })
+  @IsArray({ always: true })
+  @IsUrl({}, { always: true, each: true })
+  domains?: Set<string>;
```

Comment:
Is the type here actually a set?
I think you need something like:
`@Type(() => Set)`

Reply:
Even using the `@Type` might not work since the domain is being set outside the plainToClass
https://github.com/architect-team/architect-cli/pull/299/files#diff-de9be30d94b13b89791f6b04e91deb40f0729a7000d7e3cc1327c61a4183832eR96

Reply:
Maybe add a quick test to make sure its actually a set?